pkg/pbq/store/interfaces.go

```diff
@@ -0,0 +1,19 @@
+package store
+
+import (
+	"github.com/numaproj/numaflow/pkg/isb"
+)
+
+// Store provides methods to read, write and delete data from a durable store.
+type Store interface {
+	//ReadFromStore returns all the messages from the persisted store
+	ReadFromStore(size int64) ([]*isb.Message, error)
+	//WriteToStore writes message to persisted store
+	WriteToStore(msg *isb.Message) error
```

Comment:
should we pass an array of messages to make batch writing possible?

Reply:
store should deal with batching instead as those semantics are store specific?